core/rint/src/TRint.cxx

```diff
@@ -195,11 +195,13 @@ TRint::TRint(const char *appClassName, Int_t *argc, char **argv, void *options,
       PrintLogo(lite);
    }
 
-   // Explicitly load libMathCore it cannot be auto-loaded it when using one
-   // of its freestanding functions. Once functions can trigger autoloading we
-   // can get rid of this.
+#ifndef R__USE_CXXMODULES
+   // Explicitly load libMathCore. It cannot be auto-loaded it when using one
```

Comment:
```suggestion
   // Explicitly load libMathCore. It cannot be auto-loaded when using one
```

Reply:
Maybe one can be even more explicit. Autoloading through autoload keys in the rootmap files never worked for functions. This is why libMathCore has always been loaded automatically, given the widespread use of its free functions. However, modules fix this and we can load the library on demand, saving some time when starting ROOT (reading less files on CVMFS, for example) and saving some memory.

Reply:
done.